Cargo.toml

```diff
@@ -27,29 +27,36 @@ itertools = "0.4.1"
 time = "0.1.34"
 tempdir = "0.3.4"
 libc = "0.2.0"
+rand = "0.3.11"
+scopeguard = "0.1.2"
 
 [target.x86_64-pc-windows-gnu.dependencies]
```

Comment:
I suspect multirust only compiles on nightly, right? In that case you may want to try giving the new `[target.'cfg(windows)'.dependencies]` syntax a spin to avoid this duplication.